src/main/java/seedu/address/model/ModelManager.java

```diff
@@ -89,9 +90,29 @@ public ModelManager(ReadOnlyPropertyBook propertyBook, ReadOnlyUserPrefs userPre
         this.filteredAppointments = new FilteredList<>(this.appointmentBook.getAppointmentList());
     }
 
+    /**
+     * Initializes a ModelManager with the given appointmentBook, propertyBook and userPrefs.
+     */
+    public ModelManager(ReadOnlyAppointmentBook appointmentBook, ReadOnlyPropertyBook propertyBook,
+                        ReadOnlyUserPrefs userPrefs) {
+        super();
+        requireAllNonNull(appointmentBook, propertyBook, userPrefs);
+
+        logger.fine("Initializing with address book: " + propertyBook + " and user prefs " + userPrefs);
```

Comment:
Should update `address book` to `property book`?

Reply:
Fixed as well:)